build/includes/sdk.mk

```diff
@@ -165,7 +165,7 @@ run-sdk-conformance-test-go:
 	$(MAKE) run-sdk-conformance-no-build SDK_FOLDER=go GRPC_PORT=9001 HTTP_PORT=9101 FEATURE_GATES=PlayerTracking=true TESTS=$(DEFAULT_CONFORMANCE_TESTS),$(ALPHA_CONFORMANCE_TESTS)
 
 run-sdk-conformance-test-rust:
-	$(MAKE) run-sdk-conformance-test SDK_FOLDER=rust
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=rust FEATURE_GATES=PlayerTracking=true TESTS=$(DEFAULT_CONFORMANCE_TESTS),$(ALPHA_CONFORMANCE_TESTS)
```

Comment:
This should probably follow what Go is doing - which is running one without the flag, and then one with:
https://github.com/googleforgames/agones/blob/8b4c100070cfacb31395015acdbd2b7d19262567/build/includes/sdk.mk#L163

Reply:
Thank you for the review.
OK. As far as I can see, it seems better not to duplicate ports between tests in each language. 
I'll fix it like that.

Reply:
I fixed it.